Samples/README.md

```diff
@@ -1,13 +1,131 @@
-Please follow these below steps to run samples
-
-- Navigate to the desired sample
-- In CMD, run the command, pod install
-- Open the .xcworkspace file using xcode
-- Click on Build and run
-
-`Note`:
-In every sample, in Skyflow.Config(), replace with the following fields:
-1. Replace the placeholder "<VAULT_ID>" with the correct vaultId you want to connect
-2. Replace the placeholder "<VAULT_URL>" with the correct vaultURL
-3. Implement the bearer token endpoint using server side auth SDK and service account file.
-   Replace the placeholder "<TOKEN_END_POINT_URL>" with the  bearer token endpoint which gives the bearerToken, implemented at your backend. 
\ No newline at end of file
+# iOS SDK samples
+Test the SDK by adding `VAULT-ID`, `VAULT-URL`, and `SERVICE-ACCOUNT` details in the required places for each sample.
+
+### Prerequisites
+- iOS 13.0.0 or higher
+- [cocoapods](https://cocoapods.org)
+- Xcode
+- [Node.js](https://nodejs.org/en/) 10 or higher
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) 6.x.x or higher
+
+- [express.js](http://expressjs.com/en/starter/hello-world.html)
+
+
+### Create the vault
+1.  In a browser, sign in to Skyflow Studio.
+2. Create a vault by clicking **Create Vault** > **Start With a Template** > **PIIData**.
+3. Once the vault is ready, click the gear icon and select **Edit Vault Details**.
+4. Note your **Vault URL** and **Vault ID** values, then click Cancel. You'll need these later.
+
+### Create a service account
+1. In the side navigation click, **IAM** > **Service Accounts** > **New Service Account**.
+2. For **Name**, enter "SDK Samples". For Roles, choose **Vault Editor**.
+3. Click **Create**. Your browser downloads a **credentials.json** file. Keep this file secure. You'll need it to generate bearer tokens.
+
+### Create a bearer token generation endpoint
```

Comment:
Why don't we create and ship the files for this sample for them instead of forcing them to make the directory and files themselves?

Reply:
Because credentials file will not be in the `Samples folder`. Here we are using `credentials.json` file for generating bearer token. Which the user need to keep in the separate folder created `bearer-token-generator` and this folder will not be in this repo since it is completely node project which is used for generaton of `endpoint of token generator`.